Local workflows get stuck with `cpu` setting specified

This is one of the causes for #140 triggered by a runner panic. Local workflows can get stuck for other panic reasons.

Run a local workflow with a `cpu` setting, like this one:

```
  - name: python-version
    provider: bash
    python: 3.7
    commands:
      - python --version
    resources:
      cpu: 1
```

The CLI will get stuck on `Provisioning... It may take up to a minute.` The runner will print the following logs:

```
time=2023-01-18T10:26:24+05:00 level=info msg=Log level: info
time=2023-01-18T10:26:24+05:00 level=info msg=RUNNER START...
time=2023-01-18T10:26:35+05:00 level=error msg=[PANIC] =[backend.go:92 s3.(*S3).Init] yaml: unmarshal errors:
line 31: cannot unmarshal !!map into int
panic: [backend.go:92 s3.(*S3).Init] yaml: unmarshal errors:
line 31: cannot unmarshal !!map into int [recovered]
panic: [backend.go:92 s3.(*S3).Init] yaml: unmarshal errors:
line 31: cannot unmarshal !!map into int
```

This is because Go and Python code do not agree on the runner serialization for `cpus`. Python produces the following:

```
job:
  ...
  requirements:
    cpus:
      count: 1
```

But Go expects `cpus` as int:

https://github.com/dstackai/dstack/blob/10a2267fda9b61859de69dcbad27a2464a7e88d2/runner/internal/models/backend.go#L71-L78

--- a/cli/dstack/aws/jobs.py
+++ b/cli/dstack/aws/jobs.py
@@ -71,9 +71,7 @@ def _serialize_requirements(requirements) -> Dict[str, Any]:
     req_data = {}
     if requirements:
         if requirements.cpus:
-            req_data["cpus"] = {
-                "count": requirements.cpus
-            }
+            req_data["cpus"] = requirements.cpus
         if requirements.memory_mib:
             req_data["memory_mib"] = requirements.memory_mib
         if requirements.gpus:

